.travis.yml

```diff
@@ -83,14 +80,12 @@ deploy:
     # Push the built HTML in doc/_build/html to the gh-pages branch
     - provider: script
       script: continuous-integration/travis/deploy-gh-pages.sh
```

Comment:
Is there an alternative for these or are they the default now? This was required before so that the files we want to deploy didn’t get delete.

Reply:
They do have a key called `cleanup` now (opposite to the `skip_cleanup`) which defaults to false. Should I put that in?

Reply:
Yeah, it might be best to set it explicitly if they decide to change it later on.

Reply:
Ok, I've added back `cleanup:false`. For future reference, see https://travis-ci.community/t/skip-cleanup-true-is-now-deprecated-how-to-go-for-gradle-publish/6641.